packages/cron/project.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "cron",
+    "$schema": "../../node_modules/nx/schemas/project-schema.json",
+    "sourceRoot": "packages/cron/src",
+    "projectType": "library",
+    "targets": {
+        "build": {
+            "executor": "@nx/esbuild:esbuild",
+            "outputs": ["{options.outputPath}"],
+            "options": {
+                "outputPath": "dist/packages/cron",
+                "main": "packages/cron/src/worker.ts",
+                "tsConfig": "packages/cron/tsconfig.lib.json",
+                "assets": ["packages/cron/*.md"],
+                "generatePackageJson": true
+            }
+        },
+        "build:local": {
+            "executor": "@nx/esbuild:esbuild",
+            "outputs": ["{options.outputPath}"],
+            "options": {
+                "outputPath": "dist/packages/cron",
+                "main": "packages/cron/src/local.ts",
+                "tsConfig": "packages/cron/tsconfig.lib.json",
+                "assets": ["packages/cron/*.md"],
+                "generatePackageJson": true
```

Comment:
generatePackageJson isn't needed because we're not distributing. I'll remove this.

The assets will just include markdown files like the readme in the dist folder

Reply:
I mean it doesn't hurt that the markdown files are going to the dist folder but is there any reason to do so?

Reply:
Probably not